pkg/cmd/urfave/urfave.go

```diff
@@ -210,3 +232,66 @@ func GetGPTDocsRunner(k string, t float64, m int, e []string) (
 		GivenEvents:       e,
 	}, nil
 }
+
+func getPoliciesFromDir(policiesDir string) ([]policy.Policy, error) {
```

Comment:
```suggestion
func getPoliciesFromDir(path string) ([]policy.Policy, error) {
```

I guess that calling it name would avoid ambiguity in the code below.

Reply:
Try to convert path to the absolute path in prior stage, so we can avoid to join dir + filepath. And it's more secure.